source/libs/page-detect.js

```diff
@@ -92,3 +92,5 @@ export const isSingleCommit = () => /^commit\/[0-9a-f]{5,40}/.test(getRepoPath()
 export const isSingleFile = () => /^blob\//.test(getRepoPath());
 
 export const isTrending = () => location.pathname.startsWith('/trending');
+
+export const isUserProfile = () => document.head.querySelector('meta[property="og:type"]').attributes.content.value === 'profile';
```

Comment:
This should be URL-based. You can probably get away with `!isReserved() && !getCleanPathname().includes('/')`

Reply:
Also needs to be added to the tests file and placed in the right position (both page-detect.js and its test file are sorted by method name)